server/validate/email/validate.go

```diff
@@ -119,6 +121,31 @@ func isTemplateValid(templ *textt.Template) error {
 	}
 	return nil
 }
+type SMPTLogin struct {
```

Comment:
`SMPTLogin`  -> `SMTPLogin`

Reply:
I don't think you need to export this structure from the module. Consequently, please change the name to have a lowercase first letter. 
If you believe it needs to be exported, please explain why and add a required godoc comment.

I would name it `loginAuth` to follow the same pattern as `plainAuth`.